File: app/src/main/java/com/wimatt/ux/drinkanddare/retrofit/models/Shop.java
package com.wimatt.ux.drinkanddare.retrofit.models;

public class Shop {

    private String _id;
    private String name;
    private Address address;

    public Shop(String _id, String name, Address address) {
        this._id = _id;
        this.name = name;
        this.address = address;
    }

    public String get_id() {
        return _id;
    }

    public void set_id(String _id) {
        this._id = _id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Address getAddress() {
        return address;
    }

    public void setAddress(Address address) {
        this.address = address;
    }
}

File: app/src/main/java/com/wimatt/ux/drinkanddare/room/models/Shop.kt
package com.wimatt.ux.drinkanddare.room.models

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity
data class Shop(
        @PrimaryKey val shopId: String,

        @ColumnInfo(name = "name")
        var name: String,

        @ColumnInfo(name = "state")
        var location: String,

        @ColumnInfo(name = "address")
        var address: String
)